understanding/21/reflow.html

```diff
@@ -12,68 +12,245 @@ <h1>Understanding Reflow</h1>
+        
+        <p>However, not all content can fully reflow without degradation of the information or functionality the content represents - <a href="#note-2dlayout">as noted in the Reflow success criterion</a>. Examples are tabular data, graphics, maps, presentations, or interfaces that necessiate persistent toolbars.</p>
+
+        <h4>Understanding the scope of exceptions</h4>
+        <p>Where an exception for bi-directional scrolling would be applicable to a section of content, the exception does not extend to other sections of content which do not necessitate bi-directional scroling for understanding or functionality.</p>
+        
+        <figure>
+          <img src="img/exception-does-not-extend.jpg" alt="zoomed in screenshot of the bottom of a table and a multi-line paragraph of text. Each extend off-screen, resulting in both horizontal and vertical scrollbars at the page level.">
+          <figcaption>
+            <p>Due to the way this example was implemented, the web page has both horiztonal and vertical scrollbars. While the table meets the exception for two-dimensional layout for understanding, the paragraphs following the table in the screenshot do not.</p>
+          </figcaption>
+        </figure>
+
+        <p>Since a data table relies on two-dimensional layout for understanding, presenting the table in its own scrollable container would help ensure other content which does not meet an exception could reflow as the viewport adjusts.</p> 
+        
+        <figure>
+          <img src="img/table-exception.jpg" alt="a table within a containing element that provides bi-directional scrollbars to view the content of the table. The content above and below the table reflows to fit within the available width of the viewport.">
+          <figcaption>
+            <p>Tabular data has a Reflow exception, so a containing element to the table can provide bi-directional scrollbars, thus mitigating bi-directional scrollbars at the page level. On the left, a web page at default zoom shows a table of four columns represents the files a user has access to. On the right, the web page is zoomed in and the table is rendered within a containing element with is own bi-dimensional scrollbars.</p>
+          </figcaption>
+        </figure>
+
+        <p>There can be instances where bi-dimensional excepted content can extend beyond the visible viewport, and other non-excepted content exists, but does reflow within the visible viewport.  Unlike the prior table example with the paragraph not reflowing, such a situation does pass - but creates an unwanted user experience. For example, an unnecessary horizontal scrollbar could lead a user to believe there is content existing off-screen for them to scroll to. But if that scrollbar appears only because of one instance of excepted content, a user could expend effort looking for <em>other</em> content beyond that which reflows which doesn't exist.</p>
```

Comment:
I think "causing the entire page to scroll," while intuitive, could still be helpful here.

```suggestion
        <p>There can be instances where bi-dimensional exempted content extends beyond the visible viewport, causing the entire page to scroll. At the same time, other non-exempted content on the same page reflows within the visible viewport. Unlike the prior table example with the paragraph not reflowing, such a situation does pass - but creates an unwanted user experience. For example, an unnecessary horizontal scrollbar could lead a user to believe there is content existing off-screen for them to scroll to. But if that scrollbar appears only because of one instance of excepted content, a user could expend effort looking for <em>other</em> content beyond that which reflows which doesn't exist.</p>
```

Reply:
i don't think changing the wording from except(ed) to exempt is right?

the current understanding doc refers to these as exceptions, not exemptions.

Reply:
Edit: Updated the proposal keeping "excepted"